src/main/java/GeePee.java

```diff
@@ -0,0 +1,92 @@
+import java.util.Scanner;
+
+public class GeePee {
+
+    //constant to determine required padding to extract the name of a Todo object
+    public static final int TODO_PADDING = 5;
+
+    //constants to determine required padding to extract the name of a Deadline object,
+    //and when it needs to be completed by
+    public static final int DEADLINE_PADDING = 9;
+    public static final int BY_PADDING = 4;
+
+    //constants to determine required padding to extract the name of a Event object,
+    //and when it starts and ends
+    public static final int EVENT_PADDING = 6;
+    public static final int FROM_PADDING = 6;
+    public static final int TO_PADDING = 4;
+
+    public static void handleTodo(List list, String line) {
+        String todoName = line.substring(TODO_PADDING).trim();
+        list.addTodo(todoName);
+    }
+
+    public static void handleDeadline(List list, String line) {
+        int byIndex = line.indexOf("/");
+        String by = line.substring(byIndex + BY_PADDING).trim();
+        String deadlineName = line.substring(DEADLINE_PADDING, byIndex).trim();
+        list.addDeadline(deadlineName, by);
+    }
+
+    public static void handleEvent(List list, String line) {
+        int fromIndex = line.indexOf("/");
+        int toIndex = line.indexOf("/", fromIndex + 1);
+        String from = line.substring(fromIndex + FROM_PADDING, toIndex).trim();
+        String to = line.substring(toIndex + TO_PADDING).trim();
+        String eventName = line.substring(EVENT_PADDING, fromIndex).trim();
+        list.addEvent(eventName, from, to);
+    }
+
+    public static void handleTaskStatusChange(List list, String line) {
+        String[] words = line.split(" ");
+        int number = Integer.parseInt(words[1]);
+        if (number >= 0 && number <= list.getSize()) {
+            list.changeTaskStatus(number - 1, (words[0].equals("mark") ? true : false));
+        }
+    }
+
+    public static void loop() {
```

Comment:
can potentially split this into 2 methods as loop method is rather long and deals with alot